test/factories/ProposalFactory.t.sol

```diff
@@ -74,82 +74,83 @@ contract ProposalFactoryTest is Test {
         factory = new ProposalFactory(address(target), address(moduleFactory));
     }
 
-    function testValidProposalId(uint getId, uint proposalsCreated) public {
-        // Note to stay reasonable
-        proposalsCreated = bound(proposalsCreated, 0, 50);
-
-        for (uint i = 0; i < proposalsCreated; ++i) {
-            _deployProposal();
-        }
-        if (getId > proposalsCreated) {
-            vm.expectRevert(
-                IProposalFactory.ProposalFactory__InvalidId.selector
-            );
-        }
-        factory.getProposalByID(getId);
-    }
-
-    function testDeploymentInvariants() public {
-        assertEq(factory.target(), address(target));
-        assertEq(factory.moduleFactory(), address(moduleFactory));
-    }
-
-    function testCreateProposal(uint modulesLen) public {
-        // Note to stay reasonable
-        modulesLen = bound(modulesLen, 0, 50);
-
-        // Create optional ModuleConfig instances.
-        IProposalFactory.ModuleConfig[] memory moduleConfigs =
-        new IProposalFactory.ModuleConfig[](
-                modulesLen
-            );
-        for (uint i; i < modulesLen; ++i) {
-            moduleConfigs[i] = moduleConfig;
-        }
-
-        // Deploy Proposal with id=1
-        IProposal proposal = factory.createProposal(
-            proposalConfig,
-            fundingManagerConfig,
-            authorizerConfig,
-            paymentProcessorConfig,
-            moduleConfigs
-        );
-
-        // Check that proposal's strorage correctly initialized.
-        assertEq(proposal.proposalId(), 1);
-        assertEq(address(proposal.token()), address(proposalConfig.token));
-        assertTrue(address(proposal.authorizer()) != address(0));
-        assertTrue(address(proposal.paymentProcessor()) != address(0));
-
-        // Check that other proposal's dependencies correctly initialized.
-        // Ownable:
-        assertEq(proposal.manager(), address(proposalConfig.owner));
-
-        // Deploy Proposal with id=2
-        proposal = factory.createProposal(
-            proposalConfig,
-            fundingManagerConfig,
-            authorizerConfig,
-            paymentProcessorConfig,
-            moduleConfigs
-        );
-        // Only check that proposal's id is correct.
-        assertEq(proposal.proposalId(), 2);
-
-        //check that proposalFactory idCounter is correct.
-        assertEq(factory.getProposalIDCounter(), 2);
-    }
-
-    function testProposalMapping(uint proposalAmount) public {
-        // Note to stay reasonable
-        proposalAmount = bound(proposalAmount, 0, 50);
-
-        for (uint i = 1; i < proposalAmount; ++i) {
-            address proposal = _deployProposal();
-            assertEq(proposal, factory.getProposalByID(i));
-        }
-    }
+    // function testValidProposalId(uint getId, uint proposalsCreated) public {
```

Comment:
ModuleFactoryMock actually doesnt create any real modules so it cant call init2 functions in there, thats why it fails